tests/ensure_provider_tests.py

```diff
@@ -88,6 +88,9 @@
     "playbooks/tests_wireless.yml": {
         EXTRA_RUN_CONDITION: "ansible_distribution_major_version == '7'",
     },
+    "playbooks/tests_wireless_wpa3_owe.yml": {
+        EXTRA_RUN_CONDITION: "ansible_distribution_major_version != '7' and ansible_distribution_major_version != '32'",
```

Comment:
How about `== 8 or >= 33`

Reply:
updated

Reply:
but CI failed on Centos 7, I tested locally, got the following error:
```
TASK [Create hostapd config] *************************************************************************************************************************************************************
task path: /home/wenliang/network/tests/playbooks/tasks/setup_mock_wifi_wpa3_owe.yml:62
fatal: [CentOS-7-x86_64-GenericCloud-2009.qcow2]: FAILED! => {"changed": false, "checksum": "54fa75906e299d11baaf2abccdf7a54d69ea8b95", "msg": "Destination directory /etc/hostapd does not exist"}
```